src/apis/redeem.ts

```diff
@@ -83,6 +83,12 @@ export class DefaultRedeemAPI {
         return redeemRequests.map((v) => v[1]);
     }
 
+    async listForUser(account: AccountId): Promise<[H256, Redeem][]> {
```

Comment:
The return type here should be a `RedeemRequest`:https://github.com/interlay/polkabtc-js/blob/dev/src/interfaces/definitions.ts#L140